inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_ref.cpp

```diff
@@ -54,4 +54,24 @@ bool EltwiseKernelRef::Validate(const Params& p, const optional_params& o) const
 KernelsData EltwiseKernelRef::GetKernelsData(const Params& params, const optional_params& options) const {
     return GetCommonKernelsData(params, options);
 }
+
+JitConstants EltwiseKernelRef::GetJitConstants(const eltwise_params& params) const {
+    auto jit = EltwiseKernelBase::GetJitConstants(params);
+
+    if (!params.fused_ops.empty()) {
+        kernel_selector::Datatype input_dt = GetAccumulatorType(params);
+
+        std::vector<std::string> idx_order;
+        if (DataTensor::ChannelsCount(params.output.GetLayout()) == 4) {
+            idx_order = {"d4", "d3", "d2", "d1"};
+        } else if (DataTensor::ChannelsCount(params.output.GetLayout()) == 5) {
+            idx_order = {"d5", "d4", "d3", "d2", "d1"};
+        }
```

Comment:
Reference kernel also supports 6D layouts, so **bfwzyx** case should be handled too